torch/csrc/inductor/aot_runtime/interface.h

```diff
@@ -52,9 +53,13 @@ AOTInductorError AOTInductorModelContainerDelete(
 // Runs the inference.
 AOTInductorError AOTInductorModelContainerRun(
     AOTInductorModelContainerHandle container_handle,
-    AOTInductorTensorHandle input_handles,
+    // array of raw AtenTensorHandle for input tensors, and will be stolen by
+    // RAIIAtenTensorHandle
```

Comment:
That it gets stolen by RAIIAtenTensorHandle is an unnecessary implementation detail that doesn't need to be in the public API contract. The public API contract is that this steals all the input handles, end of story.